### PR TITLE
chore(CI): run Slack action on failures

### PR DESCRIPTION
**Pull request by tujoworker**
Ensure we run the Slack action when an action fails.

### CODE REVIEW

File: .github/workflows/visual-test.yml [hunk 1]
```diff
@@ -85,4 +85,4 @@ jobs:
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
           MATRIX_CONTEXT: ${{ toJson(matrix) }}
-        if: github.repository_owner == 'dnbexperience'
+        if: ${{ always() && github.repository_owner == 'dnbexperience' }}
```
Comment (dinarosv) [inline .github/workflows/visual-test.yml:88]: What does the `always()` do? In an if-check it seems like it should always be true?

**Review by dinarosv — APPROVED** [review]
⭐ ⭐

File: .github/workflows/visual-test.yml (hunk 1 above)
Comment (tujoworker) [reply .github/workflows/visual-test.yml:88]: yeah, it looks wired and illogical. But that's how it works:

> Causes the step to always execute, and returns true, even when canceled.

https://docs.github.com/en/actions/learn-github-actions/expressions#always

**Pull request merged**